### PR TITLE
APP-283: Review Gradle integration - Cell monitor app update

### PR DESCRIPTION
**Pull request by iamtodor**
Modifier cell monitor app

### CODE REVIEW

**Comment by kaa-leeroy** [comment]
Can one of the admins verify this patch?

**Comment by Slash32** [comment]
@kaa-leeroy add to whitelist

**Comment by vadimol** [comment]
@kaa-leeroy retest this please

File: cellmonitor/source/android/app/src/main/java/org/kaaproject/kaa/demo/cellmonitor/util/LocationUtil.java [hunk 1]
```diff
@@ -39,13 +39,13 @@ public static boolean isLocationEnabled(Context context) {
         try {
             gpsEnabled = lm.isProviderEnabled(LocationManager.GPS_PROVIDER);
         } catch (Exception ex) {
-            ex.printStackTrace();
+            CellMonitorConstants.LOG.error("GPS provider disabled");
```
Comment (maxml) [inline cellmonitor/source/android/app/src/main/java/org/kaaproject/kaa/demo/cellmonitor/util/LocationUtil.java:42]: ex.printStackTrace(); - add it back

**Pull request merged**